documentation/docs.css

```diff
@@ -0,0 +1,189 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+body{
+    font-family: 'Montserrat', sans-serif;
+}
```

Comment:
Suggestion: Add scroll-behavior: smooth;

Reply:
Done 
Thanks for the suggestion